Extensions/TextInput/textinputruntimeobject-pixi-renderer.ts

```diff
@@ -210,12 +228,14 @@ namespace gdjs {
       const widthInContainer = pageRight - pageLeft;
       const heightInContainer = pageBottom - pageTop;
 
-      this._input.style.left = pageLeft + 'px';
-      this._input.style.top = pageTop + 'px';
-      this._input.style.width = widthInContainer + 'px';
-      this._input.style.height = heightInContainer + 'px';
-      this._input.style.transform =
+      this._form.style.left = pageLeft + 'px';
+      this._form.style.top = pageTop + 'px';
+      this._form.style.width = widthInContainer + 'px';
+      this._form.style.height = heightInContainer + 'px';
+      this._form.style.transform =
         'rotate3d(0,0,1,' + (this._object.getAngle() % 360) + 'deg)';
+      this._input.style.padding = this._object.getPadding() + 'px';
```

Comment:
I think the padding should have a minimum of 0, otherwise you might have some weird results

Reply:
Maybe it should be handled in the method `getPadding` directly